client/sites/tools/deployments/style.scss

```diff
@@ -8,4 +8,9 @@
 		align-self: flex-start;
 		margin: 0;
 	}
+
+	.components-button.is-primary.is-busy {
```

Comment:
Should we keep the default gray linear gradient and align that with the designers?

Reply:
I'm not sure I follow. I'm adding these styles to revert to the Gutenberg styles because we're adding our own global overrides.

| Before | After |
| ----- | ------- | 
| https://github.com/user-attachments/assets/32dbe076-07e1-42e3-8bbd-3580c525b30c | https://github.com/user-attachments/assets/2a5f9721-13f9-4a41-ad63-3e7a3cbc99b6 |

Reply:
I'm not sure I follow. I'm adding these styles to revert to the Gutenberg styles because we're adding our own global overrides.

Here's what we have with this patch:

https://github.com/user-attachments/assets/c68d9af1-a460-45e6-beb9-f3eb8982a0b3

And here's what we get if we remove it:

https://github.com/user-attachments/assets/d115c258-7c39-4e2b-8d08-95a2e4ca414e

Is there a better way of achieving the same result?

Reply:
> I'm adding these styles to revert to the Gutenberg styles because we're adding our own global overrides

I mean, maybe we should search and remove the overrides instead of adding another one since we've been working towards removing any Gutenberg overrides recently.

Reply:
Absolutely. I'm just not sure how we can do it. But I'll take a look.

Reply:
I tried looking for a solution but it seems that it's a bit more complex than that. I ended up adding a FIXME comment pointing to this Slack message: p1731590534067879-slack-C03DY3RJ5. It's totally unrelated to the PR so that's why I think we should address it elsewhere (and remove the patches.)